src/pages/ai-creation/pages/shikshalokam-mitra/mitra-pages/components/Guidelines.jsx

```
@@ -0,0 +1,35 @@
+      onMouseEnter={() => setShowTooltip(true)}
+      onMouseLeave={() => setShowTooltip(false)}
+    >
+      <div className="flex items-center gap-1 cursor-pointer text-[#666666] hover:text-[#1177FF] transition-colors">
+        <span className="text-sm font-medium">Guidelines</span>
+        <IoInformationCircleOutline className="w-4 h-4" />
+      </div>
+
+      {showTooltip && (
+        <div className="absolute left-0 top-full pt-2 z-50">
+          <div className="relative w-64 md:w-80 p-3 bg-white border border-[#DDDDDD] rounded-lg shadow-lg max-h-60 overflow-y-auto">
+            <p className="text-sm text-[#333333] leading-relaxed whitespace-pre-wrap" dangerouslySetInnerHTML={{ __html: text }}>
```

Biome (2.1.2)

[error] Avoid passing both children and the dangerouslySetInnerHTML prop.

This is the source of the children prop

Setting HTML content will inadvertently override any passed children in React

(lint/security/noDangerouslySetInnerHtmlWithChildren)